lib/src/set/set_builder.dart

```diff
@@ -12,6 +12,8 @@ part of built_collection.set;
 /// [Built Collection library documentation](#built_collection/built_collection)
 /// for the general properties of Built Collections.
 class SetBuilder<E> {
+  /// Used by [_createSet] to instantiate [_set]. The default value is `null`.
+  Set<E> Function() _setConstructor;
```

Comment:
I think "_setFactory" would be a better name; "constructor" refers to, well, constructors :)